lib/meta_tags/text_normalizer.rb

```diff
@@ -71,7 +71,11 @@ def self.helpers
     # @return [String] string with no HTML tags.
     #
     def self.strip_tags(string)
-      ERB::Util.html_escape helpers.strip_tags(string)
+      # TODO: Due to a bug in strip_tags (introduced with Rails 4.2), ampersands
+      #       and quotes are escaped twice. The "to_s.gsub" can be removed
+      #       once the affected Rails versions are not widely used anymore, see:
+      #       https://github.com/rails/rails-html-sanitizer/issues/28   
+      ERB::Util.html_escape helpers.strip_tags(string).to_s.gsub(/&amp;/, '&')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.